### PR TITLE
 fix: Additional alias names of custom emoji accepted

### LINKED ISSUE #18264: Additional alias names of custom emojis not accepted anymore
### Description:

[According to our docs](https://docs.rocket.chat/guides/administrator-guides/custom-emoji), it should be possible to add multiple alias names to custom emojis, separated by space character: 

> Enter a name for your emoji, add space separated list of the aliases for the emoji which can be entered to display the emoji in the chats

This does not seem to be working anymore, the space character will be replaced by a dash instead.

### Steps to reproduce:

1. Add a new custom emoji
2. Give it multiple aliases, e.g. `first second`
3. Hit save, see how the alias gets transformed into: `first_second` 

### Expected behavior:

It should separate the alias input by space, as stated in the docs, and accept multiple alias names.

### Actual behavior:

Space character gets replaced by dash. Multiple aliases not possible.

### Server Setup Information:

- Version of Rocket.Chat Server: 3.4.0
- Operating System: Linux
- Deployment Method: Docker
- Number of Running Instances: Docker 
- DB Replicaset Oplog: Yes
- NodeJS Version: Docker
- MongoDB Version: Docker

##### Demo video

[Bildschirmfoto 2020-07-15 um 10 45 26](https://up.frd.mn/0zWo1s8hFd.mp4)

### PR DESCRIPTION
**Pull request by anikdhabal**
## Proposed changes (including videos or screenshots)

Before:-
Screenshot 2023-04-18 184309
After:-
Screenshot 2023-04-18 184658

## Issue(s)

closes #18264 
## Steps to test or reproduce

1.Add a new custom emoji
2.Give it multiple aliases, e.g. X Y
3.Hit save, see how the alias gets transformed into: X_Y

### CODE REVIEW

**Comment by debdutdeb** [comment]
could you summarize the change, please?

**Comment by anikdhabal** [comment]
@debdutdeb The problem is that Space character gets replaced by underscore. It's done by the limax module to replace space with underscore. And this line **split(/[\s,]/)** split the alias string into an array at each occurrence of whitespace. For that reason whitespace is added.

**Comment by debdutdeb** [comment]
> And this line split(/[\s,]/) split the alias string into an array at each occurrence of whitespace. For that reason whitespace is added. So, I just remove it.

But we need to split by whitespace, without it how would `this that` get to be two aliases instead of one?

**Comment by anikdhabal** [comment]
> > And this line split(/[\s,]/) split the alias string into an array at each occurrence of whitespace. For that reason whitespace is added. So, I just remove it.
> 
> But we need to split by whitespace, without it how would `this that` get to be two aliases instead of one?
> 
> am I following the logic correctly?

Yes Yes Understood.

File: apps/meteor/app/emoji-custom/server/methods/insertOrUpdateEmoji.ts [hunk 1]
```diff
@@ -66,7 +66,7 @@ Meteor.methods<ServerMethods>({
 					field: 'Alias_Set',
 				});
 			}
-			aliases = _.without(emojiData.aliases.split(/[\s,]/).filter(Boolean), emojiData.name);
+			aliases = _.without(emojiData.aliases.split(/[,]/).filter(Boolean), emojiData.name);
```
Comment (debdutdeb) [inline apps/meteor/app/emoji-custom/server/methods/insertOrUpdateEmoji.ts:69]:
```suggestion
			aliases = _.without(emojiData.aliases.split(/[\s,]/).filter(Boolean), emojiData.name);
```

**Review by debdutdeb — CHANGES_REQUESTED** [review]

**Review by debdutdeb — APPROVED** [review]
lgtm

**Comment by ggazzo** [comment]
https://github.com/RocketChat/Rocket.Chat/issues/18264#issuecomment-3962147588
